src/main/java/ticTacToe/game/Mark.java

```diff
@@ -2,18 +2,36 @@
 
 public enum Mark {
 
-    NOUGHT("O"), CROSS("X");
+    NOUGHT("O"),
+    CROSS("X");
 
-    Mark(String name) {
-        mark = name;
+    Mark(String mark) {
+        sign = mark;
    }
 
-    public String mark;
+    public final String sign;
 
-    public static String switchPlayerMark(String currentMark) {
-        if (currentMark.equals(CROSS.mark)) {
-            return NOUGHT.mark;
+    public static boolean isValidMark(String markChosen) {
+        for (Mark mark : Mark.values()) {
+            if (mark.sign.equals(markChosen)) {
+                return true;
+            }
         }
-        return CROSS.mark;
+        return false;
+    }
+
+    public static Mark createMark(String markChosen) {
+        if (markChosen.equals(NOUGHT.sign)) {
+            return NOUGHT;
+        } else {
+            return CROSS;
+        }
+    }
+
+    public static Mark switchPlayerMark(Mark currentMark) {
```

Comment:
this could made simpler by using an instance method on the enum like so:
```java
public Mark switch() {
  if (this == CROSS) {
    return NOUGHT;
  } else {
    return CROSS;
  }
}
```

Reply:
Tried quickly to implement this but doesn't work. I might need help here